doc/source/api/index.rst

```diff
@@ -27,6 +28,7 @@ The following API documentation describes how to use Covalent.
 - :ref:`file_transfer` is used to queue remote or local file transfer operations prior or post electron execution.
 - :ref:`file_transfer_strategies` are used to perform download/upload/copy operations over various protocols.
 - :ref:`dask_executor` is used to execute electrons in a Dask cluster
+- :ref:`triggers` are used to execute a saved workflow depending on occurrence of a desired event.
```

Comment:
I think yours is better, `are used to execute a workflow triggered by a specific event type.`. I don't like "undispatched" because we do call `ct.dispatch` on the workflow.